docs/Containerfile.5.md

```diff
@@ -606,6 +606,38 @@ The secret needs to be passed to the build using the --secret flag. The final im
   $ podman build --build-arg HTTPS_PROXY=https://my-proxy.example.com .
   ```
 
+Platform/OS/Arch **ARG**
+   -- ARG <name>
+
+  Additional semiautomatic predefined `ARG` variables are available to
+  assist when producing multi-arch manifest-lists or foreign-architecture
+  images.  The purpose of each of of these variables should mostly be
+  evident from their names.  Where the "BUILD" prefix signifies details
+  about the host performing the build, and the "TARGET" prefix represents
+  the intended platform for the content.
+
+  Each of the following ARG variables must be defined empty before use,
```

Comment:
It's a really dumb (Docker) design.  You literally put (for example) `ARG TARGETARCH` in your `Containerfile`.  Forever after that, you can reference `${TARGETARCH}` and it will hold the value (for example) `amd64` or `ppc64le` depending on what platform the build is happening for.